pages/firststeps.md

```diff
@@ -45,6 +45,8 @@ Review these [Vagrant instructions](vagrant.md) to ensure that you have fully co
 
 Follow the instructions on the [GitHub and Markdown page](githubandmarkdown.md). Make sure that you've linked to your github.io and your personal .md page on the [Gitter chat](https://gitter.im/open-learning-exchange/chat) (&lt;username&gt;.github.io and &lt;username&gt;.github.io/#!pages/profiles/&lt;username&gt;.md).
 
+[Markdown Tutorial](https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet#links)
```

Comment:
Hello.
I got removed the line in firststeps.md.
I added the tutorial's link in "githubandmarkdown.md"
Could you take a look please?
Could you please approve this pull request please?